msteams-platform/whats-new.md

```diff
@@ -7,8 +7,7 @@ keywords: teams what's new latest
 ---
 # What's new for developers in Microsoft Teams
 
->[!TIP]
-> Check out sample solutions to common team scenarios in the [**Teams app template catalog**](samples/app-templates.md)! The catalog is alphabetized and the newest additions are tagged with a star ☆.
+Find out the new features in Microsoft Teams that are generally available (GA) or are in developer preview. You can also share your feedback. 
```

Comment:
```suggestion
Discover Microsoft Teams platform features that are generally available (GA) and in developer preview.
```
Edited for conciseness. I don't think we have to discuss sharing feedback here since it isn't the primary purpose of the topic.